README.md

```diff
@@ -1,981 +1,19 @@
-[![Coverage Status](https://coveralls.io/repos/github/containerbuildsystem/cachito/badge.svg?branch=master)](https://coveralls.io/github/containerbuildsystem/cachito?branch=master)
+# Cachi2
 
-# Cachito
+Cachi2 is a tool that identifies, fetches and lists the dependencies needed to perform a container build in a network-isolated environment. In order to do that, Cachi2 needs to be pointed to previously cloned source code and have a package manager (such as pip or go mod) specified. It will then proceed to fetch all dependencies that are declared and make them available in a local output folder, so they can be used by a following build step.
 
-Cachito is a service to store (and serve) source code for applications. Upon a request, Cachito
-will fetch a specific revision of a given repository from the Internet and store it permanently in
-its internal storage. Namely, it stores the source code for a specific Git commit from a given Git
-repository, which could be from a forge such as [GitHub](https://github.com) or
-[GitLab](https://gitlab.com). This way, even if that repository (or that revision) is deleted, it
-is still possible to track the pristine source code for the original sources. In fact, if the
-sources have already been previously fetched, Cachito will simply serve the stored copy.
+Cachi2 also generates a content manifest that contains all the dependencies that were fetched and their exact versions, serving as provenance of the build. It can also list transitive dependencies that are not directly listed in the source code, but that end up being part of the built container image.
```

Comment:
What do you mean by the transitive dependencies? For most package managers, Cachito only lists what is present in the lockfile (or requirements.txt, which we force to be a lockfile)

I could see this applying to gomod, although from 1.17 onwards I believe the go.mod file also lists everything, including the transitive ones?

Reply:
Yeah, I had gomod in mind. But you're right, Go 1.17 gets rid of that, let me fix the readme.